bpf-prog/ipv6-hp-bpf/linux-arm64.Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM mcr.microsoft.com/oss/go/microsoft/golang:1.21 AS builder
```

Comment:
Why two different Dockerfiles? You can specify `Platform` in docker and it will pull the image with specified ARCH. If for arm64, you need to run additional commands, you can add an if-else in the docker file.

Here's Retina's Dockerfiles - https://github.com/microsoft/retina/blob/main/controller/Dockerfile

We use this one Dockerfile to generate both and and arm images.

Reply:
Discussed offline; here's the suggestions:
- consolidate two docker files into 1
- Use `Arg ARCH` to branch the steps in builder
- Move the copy lines to builder, use a common location like `/tmp/req/lib/...`
- One copy line in final image to copy `/tmp/req` to `/lib`

Reply:
Thanks! I was able to consolidate back down to one docker file using the tmp directory